app/assets/src/components/views/compare/samples_heatmap_vis.scss

```diff
@@ -18,13 +19,15 @@
   }
 
   .heatmapContainer {
-    overflow-x: auto;
-    // Set the height to force the horizontal scrollbar to appear at the bottom
-    // of the viewport instead of the bottom of the page. 334px is the height
-    // of the all the elements above the heatmapContainer. It *will* need to
-    // change if the height of the elements above ever change.
-    // TODO(mark): Use flexbox instead of hard-coding the offset.
-    height: calc(100vh - 334px);
+    overflow: hidden;
+    // Set the height so the heatmap takes up the rest of the screen.
+    // 358px is the height of the elements above the heatmapContainer.
+    // It *will* need to change if the height of the elements above ever change.
+    height: calc(100vh - 358px);
+
+    &.fullScreen {
+      height: calc(100vh - 85px);
```

Comment:
I notice a tiny gap at the bottom of the screen. Is that intentional? If not, could you increase the height to the largest value before a scrollbar appears?